PackageExplorer/PublishPackageWindow.xaml

```diff
@@ -63,6 +63,26 @@
             <TextBox x:Name="PublishKey" VerticalContentAlignment="Center" IsEnabled="{Binding CanPublish}" Margin="0" Text="{Binding PublishKey}">
             </TextBox>
         </DockPanel>
+        <!-- Publish Key type-->
+        <DockPanel DockPanel.Dock="Top" Margin="7,0,10,5">
+            <Label DockPanel.Dock="Left" Content="Key type:" Width="75" />
+            <RadioButton x:Name="UseApiKey" 
+                         GroupName="KeyType" 
+                         IsEnabled="{Binding CanPublish}" 
+                         VerticalAlignment="Center" 
+                         IsChecked="{Binding UseApiKey}" 
+                         Content="API Key" 
+                         ToolTip="Use this when only an API Key is required"
+                         />
+            <RadioButton x:Name="UsePAT" 
```

Comment:
I prefer to use the term "AccessToken" instead  of PAT as we also have "UseAccessToken"

Reply:
I'll update that

Reply:
the VSTS UI uses "Personal access token" 
![image](https://user-images.githubusercontent.com/1427284/33408505-ddccfbce-d544-11e7-9e86-7cc684a4e4d1.png)

Reply:
One thing that i noticed is that i'm using VSTS as a reference. 
VSTS doesn't require a username, just the Personal Access Token as a password in the credentials.
Right now i'm sending "PAT" in the username.
Maybe we should let the user specify actual credentials? Just in case, we are limiting the scope for VSTS and missing others.

Reply:
> Maybe we should let the user specify actual credentials? Just in case, we are limiting the scope for VSTS and missing others.

that is indeed better :)